### PR TITLE
Made some changes to product proposal template (comments)

### PR DESCRIPTION
**Pull request by nlharris**
Changed (softened) some language in response to Matt H's comments. Also adding new field for Keywords in response to Roy's suggestion.

### CODE REVIEW

File: Product proposal template.md [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
 
-These high level descriptions must include the following:
+For most types of products, product descriptions should include the following:
 
 ### Summary
-A brief summary of what the product will enable or improve for the user.
+A brief summary of what the product will enable or improve for the user. (In other words, WHY should we develop this product?)
+
+### Keyword(s)
+What areas of science or technology does this product support?
```
Comment (RoyKBase) [inline Product proposal template.md:11]: Think of this as analogous to a "tag" - what parts of KBase (ideally the roadmap) does this affect?

File: Product proposal template.md [hunk 2]
```diff
@@ -1,30 +1,33 @@
...
 
-In addition to the below, proposed KBase apps should also follow the requirements in the [KBase product requirements document](./KBase%20product%20requirements.md)
+*Proposed KBase apps should also follow the requirements in the [KBase product requirements document](./KBase%20product%20requirements.md). Products that are not apps should follow those requirements where appropriate.*
 
-These high level descriptions must include the following:
+For most types of products, product descriptions should include the following:
 
 ### Summary
-A brief summary of what the product will enable or improve for the user.
+A brief summary of what the product will enable or improve for the user. (In other words, WHY should we develop this product?)
```
Comment (RoyKBase) [inline Product proposal template.md:8]: The why is essential as it helps to assess the value provided by the product

File: Product proposal template.md [hunk 3]
```diff
@@ -1,30 +1,33 @@
...
   3. If the new functionality requires management or analysis of data, then exemplar data sets and their logic for inclusion and testing must be made available and uploaded if possible into the mock-narrative.
   4. If the new functionality is improved experience, then the Narrative should be "annotated" to describe how the platform improvement would improve that experience. Many times the improvement might not directly impact the Narrative experience and that is ok as long as the improvement is described in the Narrative.
 
 ### Timeline
-  A timeline should include partial, stepwise goals with a rough timeline. For example, what partial functionality or improvements would be available at the end of each sprint.
+  A timeline should include partial, stepwise goals with a rough timeline. For example, what partial functionality or improvements would be available at the end of each sprint. Please avoid excessively detailed implementation plans in this document, though some implementation plans may be needed in order to specify a timeline.
```
Comment (RoyKBase) [inline Product proposal template.md]: I would be ok with ballpark estimates; for example, this would take 3 sprints for full functionality

**Comment by nlharris** [comment]
Roy, did you want to add your explanations to the text of this template, or were they more "meta" comments directed primarily at those who are evaluating this pull request?

**Pull request merged**